css/main.css

```diff
@@ -16,9 +20,14 @@
   --hover-color: rgb(212, 250, 250);
   --anchor-el-color: #000000;
   --footer-color: #fff;
+  --warn-color:rgb(255, 150, 14);
+  --danger-color:red;
+  --success-color:palegreen;
+
   --pale-white:#fff;
   --badge-color:#ff5677;
   --hero-color:#282c3f;
+  --grey-text:#C0A080;
```

Comment:
same suggestion: variables can be added

Reply:
these are variables only and i'm using this in other files